File: YIU-AI-UI/src/models/council.ts
export interface Council {
  name: string;
  link: string;
  year: number;
  slogan: string;
  thumbnails: ImageProps[];
  people: ImageProps[];
  description: string;
}

export interface CouncilEvent {
  id: number;
  title: string;
  description: string;
}

export interface ImageProps {
    id: number,
    type: string,
    typeId: number,
    category: string,
    originName: string,
    saveName: string,
    size: number,
    createdAt: string,
}

File: YIU-AI-Server/src/main/java/devdragons/yiuServer/domain/Council.java
package devdragons.yiuServer.domain;

import jakarta.persistence.*;
import lombok.*;
import org.hibernate.annotations.CreationTimestamp;
import org.hibernate.annotations.UpdateTimestamp;

import java.time.LocalDateTime;

@Entity
@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor(access = AccessLevel.PROTECTED)
public class Council {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(unique = true)
    private Integer id;

    @Column(nullable = false)
    private String name;

    @Column(nullable = false)
    private String link;

    @Column(nullable = false)
    private Integer year;

    @Column(nullable = false, length = 255)
    private String slogan;

    @Column(columnDefinition = "TEXT", nullable = false)
    private String description;

    @CreationTimestamp
    @Column
    private LocalDateTime createdAt;

    @UpdateTimestamp
    @Column
    private LocalDateTime updatedAt;
}
